native/src/modules/endpoint/reducers/pushCategory.js

```diff
@@ -46,6 +46,12 @@ const pushCategory = (state: CityContentStateType, action: PushCategoryActionTyp
     }
   }
 
+  // Check whether another page in the same city is loading, e.g. because it is being refreshed.
+  // This is important for displaying the loading spinner.
```

Comment:
Why is this important for showing the loading spinner?

Reply:
Because otherwise no loading spinner is shown as the current route would be 'ready'